02_bubble/pfannerj/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var mainout io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintf(mainout, "Bubble sort starting...\n")
```

Comment:
Please moving test cases from main to test files.

Reply:
Sorry, I don't know what u mean here??

Reply:
If you use table driven test cases, you won't be needing calling the sort functions many times in main(). Everything is done in the test file.

Reply:
They are just value-add comments that I decided to include in the main so that it logs what it is doing... they're not for the test cases - they'll be written when it runs just with 'go run main.go' and are quite helpful to be able to see what it is doing... but it's not in the 'requirements'